omrsigcompat/makefile

```diff
@@ -25,6 +25,20 @@ OBJECTS := omrsig
 OBJECTS := $(addsuffix $(OBJEXT),$(OBJECTS))
 EXPORT_FUNCTIONS_FILE := omrsig.exportlist
 
+ifeq (win,$(OMR_HOST_OS))
+COPYRIGHT_RC_FILE := ../../omrsigcompat/omrsig.rc
+COPYRIGHT_RES_FILE := omrsig.res
```

Comment:
It won't fail because the dependency checking (omrsig.dll: omrsig.res) works only when omrsig.rc exists at the specified folder. If there is no omrsig.rc,  the changes here get ignored and nothing is added to omrsig.dll.